'use server';

export const signIn = async () => {
  try {

  } catch (error) {
    console.log("signIn function failed.", error);
  }
}

export const signUp = async (userData: SignUpParams) => {
  try {

  } catch (error) {
    console.log("signIn function failed.", error);
  }
}